src/main/webapp/search-interview.js

```diff
@@ -40,8 +44,20 @@ function selectInterview(interviewer) {
       `PM with a ${interviewer.getAttribute('data-company')} ` +
       `${interviewer.getAttribute('data-job')}. Check your email for more ` +
       `information.`);
+    const request = new Request(`/scheduled-interviews?startTime=2020-07-13T18:30:10Z&endTime=2020-07-13T19:30:10Z&interviewer=${interviewer.getAttribute('data-email')}&interviewee=	120316124221414916520`, {
```

Comment:
Too long of a line

Reply:
If you're using this for testing, please don't, this is the code I'm working on rn. Directly access Datastore and add a scheduled interview to display it or else remove it now that you know it works.

Reply:
Done